examples/eq_runner_to_downstream/payload_v2/business/surveyresponse_0_0_3_with_supplementary_data.json

```diff
@@ -0,0 +1,229 @@
+  "data_version": "0.0.3",
+  "origin": "uk.gov.ons.edc.eq",
+  "collection_exercise_sid": "41834e4b-ba01-4cb0-9da5-a5da011b8abd",
+  "schema_name": "test_supplementary_data",
+  "flushed": false,
+  "submitted_at": "2023-08-23T08:17:34+00:00",
+  "launch_language_code": "en",
+  "survey_metadata": {
+    "survey_id": "123",
+    "form_type": "0253",
+    "ru_name": "ESSENTIAL ENTERPRISE LTD.",
+    "period_id": "201605",
+    "sds_dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178",
+    "user_id": "UNKNOWN",
+    "ru_ref": "12346789012A"
+  },
+  "data": {
+    "answers": [
+      {
+        "answer_id": "same-email-answer",
+        "value": "Yes"
+      },
+      {
+        "answer_id": "trading-answer",
+        "value": "1947-11-30"
+      },
+      {
+        "answer_id": "sales-bristol-answer",
+        "value": 12345
+      },
+      {
+        "answer_id": "sales-london-answer",
+        "value": 98765
+      },
+      {
+        "answer_id": "list-collector-employees-answer",
+        "value": "No"
+      },
+      {
+        "answer_id": "any-additional-employee-answer",
+        "value": "Yes"
+      },
+      {
+        "answer_id": "employee-first-name",
+        "value": "John",
+        "list_item_id": "UqlQQY"
+      },
+      {
+        "answer_id": "employee-last-name",
+        "value": "Doe",
+        "list_item_id": "UqlQQY"
+      },
+      {
+        "answer_id": "employee-first-name",
+        "value": "Jane",
+        "list_item_id": "zfqCLS"
+      },
+      {
+        "answer_id": "employee-last-name",
+        "value": "Smith",
+        "list_item_id": "zfqCLS"
+      },
+      {
+        "answer_id": "list-collector-additional-answer",
+        "value": "No"
+      },
+      {
+        "answer_id": "employment-start",
+        "value": "2000-11-30",
+        "list_item_id": "zPDHbT"
+      },
+      {
+        "answer_id": "additional-employment-start",
+        "value": "2000-11-30",
+        "list_item_id": "UqlQQY"
+      },
+      {
+        "answer_id": "additional-employment-start",
+        "value": "2000-11-30",
+        "list_item_id": "zfqCLS"
+      },
+      {
+        "answer_id": "list-collector-products-answer",
+        "value": "No"
+      },
+      {
+        "answer_id": "product-sales-answer",
+        "value": 20,
+        "list_item_id": "ywFBww"
+      },
+      {
+        "answer_id": "product-sales-answer",
+        "value": 30,
+        "list_item_id": "XtjjTB"
+      }
+    ],
+    "lists": [
+      {
+        "items": ["ywFBww", "XtjjTB"],
+        "name": "products",
+        "supplementary_data_mapping": {
+          "89929001": "ywFBww",
+          "202346331": "XtjjTB"
+        }
+      },
+      {
+        "items": ["zPDHbT"],
+        "name": "employees",
+        "supplementary_data_mapping": {
+          "429001": "zPDHbT"
+        }
+      },
+      {
+        "items": ["UqlQQY", "zfqCLS"],
+        "name": "additional-employees"
+      }
+    ],
+    "supplementary_data": {
```

Comment:
Since this repo doesn't validate this, we should make the example minimal, some top level and 1 repeating item with some data is sufficient.

Reply:
Sure, trimmed the example down whilst keeping nested data in both the top level and repeating items